static/src/components/builder/store/modules/multiselectInput.js

```diff
@@ -8,7 +8,7 @@ export function initialState () {
     labelAdded: false,
     choiceList: [firstElement],
     pybAnswer: '',
-    initialValue: '',
+    initialValue: [],
```

Comment:
Can `initialValue` hold multiple items or is this intended to hold a single item? If it should just hold a single item, perhaps an empty string or `null` would work here?

Reply:
initialValue in the Vue component is designed to hold multiple items, but for this iteration it was understood that initialValue was a single value so the UI only supports one item for the moment.